### PR TITLE
Adding upsert capability to Vendor

### PR DESCRIPTION
**Pull request by jeperkins4**
The update method does not appear to actually update the record on NetSuite.  I had to make several fields read only as well.

### CODE REVIEW

File: lib/netsuite/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Netsuite
-  VERSION = '0.7.1'
+  VERSION = '0.7.1.2'
```
Comment (iloveitaly) [inline lib/netsuite/version.rb]: @jeperkins4 Can you remove the version bump? I'll handle this when I merge the branch.

**Comment by iloveitaly** [comment]
@jeperkins4 Thanks for the PR! Do you mind squashing the changes into a single commit to keep the version history clean?

**Comment by jeperkins4** [comment]
Mike, This should be a bit cleaner.  Thanks for the review and for considering this PR!

**Comment by iloveitaly** [comment]
Perfect! Merged with 76f558f1a462949b64d527238d2a12dc35018603

**Pull request closed without merging**